swap_meet/decor.py

```diff
@@ -0,0 +1,10 @@
+from swap_meet.item import Item
+
+
+class Decor(Item):
+    def __init__(self, condition=0):
+        self.category = "Decor"
+        self.condition = condition
```

Comment:
Same comment in Clothing can be applied here

Reply:
Done!